tests/behat/features/content_type.civictheme_alert.view.feature

```diff
@@ -118,3 +120,60 @@ Feature: CivicTheme Alert content type render
     And I wait for AJAX to finish
     Then I should see the text "[TEST] Test alert body all pages"
     And I should not see the text "[TEST] Test alert body all pages future"
+
+  @api @javascript
```

Comment:
Added test for url patterns 
as api endpoint test (ensure it does not have html in it) requires https://github.com/ubirak/rest-api-behat-extension/tree/master